Source/NSGraphicsContext.m

```diff
@@ -515,6 +515,12 @@ + (void) handleExposeRect: (NSRect)rect forDriver: (void *)driver
 {
 }
 
+/* Private method for handling shadows */
+- (void) setShadow: (NSShadow *)shadow
+{
+  ASSIGN(_shadow, shadow);
```

Comment:
Most likely a RELEASE in the dealloc method is needed

Reply:
added.